docs/content/installation/using-the-jwt-token-docker-secret.md

```diff
@@ -18,51 +18,151 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
     ```
+   In the above command, it is important that the `--docker-username=<JWT Token>` contains the contents of the token and is not pointing to the token itself. Ensure that when you copy the contents of the JWT token, there are no additional characters or extra whitepaces. This can invalidate the token and cause 401 errors when trying to authenticate to the registry.
 
 2. Confirm the details of the created secret by running:
 
 	```bash
     kubectl get secret regcred --output=yaml
     ```
 
-3. You can now add this secret to a deployment spec or to a service account to apply to all deployments for a given SA spec. See the [Create a Pod that uses your Secret](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-pod-that-uses-your-secret) and [Add ImagePullSecrets to a service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account) documentation for more details.
+3. We are now going to use our newly created kubernetes secret in our `helm` and `manifest` deployments.
+
+
+### Manifest deployment
+
+[Installling with Manfiets](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/)
+
+Manifest deployment example:
+
+```yaml
+spec:
+  serviceAccountName: nginx-ingress
+  imagePullSecrets:
+  - name: regcred
+  automountServiceAccountToken: true
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+#        fsGroup: 101 #nginx
+  containers:
+  - image: private-registry.nginx.com/nginx-ic/nginx-plus-ingress:3.1.1
+    imagePullPolicy: IfNotPresent
+    name: nginx-plus-ingress
+```
+
+Notice `imagePullSecrets` and `containers.image` lines to represent our kubernetes secret as well as the registry and version of the NGINX Ingress controller we are going to deploy.
+
+### Helm install method
+
+If you are using `helm` to install, you can install using two methods. First is the `helm` sources method. [Helm sources install](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/#managing-the-chart-via-sources "helm source install")
+
+1. Clone the nginxinc/kubernetes-ingress repository
+2. Change directory into deployents/helm-chart of the recently cloned repo.
+3. Modify the `values.yaml` file.
+
+We want to set a few lines for NGINX Plus Ingress controller to be deployed.
+
+1. Change to `nginxplus`
+2. Specify NGINX Ingress controller image to use.
+3. Specify a `imagePullSecretName` to pull from the private registry.
+
+
+### Ensure nginxplus is set to `true`
+```yaml
+## Deploys the Ingress Controller for NGINX Plus.
+nginxplus: true
+```
+
+### Specify the image to be used for deployment
+
+```yaml
+image:
+  ## The image repository of the Ingress Controller.
+  repository: private-registry.nginx.com/nginx-ic/nginx-plus-ingress
+
+  ## The tag of the
+  tag: 3.1.1
+```
+
+### Specify the `imagePullSecrets`
```

Comment:
I would update this to `imagePullSecretName` as well.  Sorry I missed it the first time.